person.rb

```diff
@@ -0,0 +1,22 @@
+# Class Person file.
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] The third parameter of your constructor is not correct, actually, when you try to create a person object it expects 2 arguments instead of three. Let's fix this.

Hint: `parent_permission = true` This default the value of the parameter to `true` if the value is not passed when creating a new Person.

Reply:
@EricMbouwe I have done that before, but rubocop fails with that.
<img width="1269" alt="Screen Shot 2021-12-13 at 7 19 33 PM" src="https://user-images.githubusercontent.com/24655018/145897959-28ad71a7-1faa-456a-b700-7d3963949091.png">

Reply:
@EricMbouwe 
<img width="1269" alt="Screen Shot 2021-12-13 at 7 28 28 PM" src="https://user-images.githubusercontent.com/24655018/145899122-2b98ebc8-518e-43d0-940f-1a09fd40500f.png">

Reply:
Hi @ramiaquista,
I have double-checked and yes you can use that syntax recommended by the linter. Kindly use your previous code for this piece.

Reply:
Ok! 😄